pkgs/development/tools/boost-build/default.nix

```
@@ -39,6 +39,8 @@ stdenv.mkDerivation {
   postPatch = ''
```

@wegank What are your thoughts on this?

```suggestion
  postPatch = lib.optionalString stdenv.isDarwin ''
```

---

I prefer to keep patches as general as possible, so I'm not encouraging this.

---

Thanks for the helpful clarification 🙂